buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1762,14 +1762,17 @@ class BeamModulePlugin implements Plugin<Project> {
         }
 
         if (runner?.equalsIgnoreCase('direct')) {
+          testRuntimeOnly it.project(path: ":runners:direct-java")
```

Comment:
```suggestion
```

This one shouldn't be needed since shadowTest should contain the main jar as well.